warehouse/query-core/src/main/java/datawave/query/planner/rules/NodeTransformVisitor.java

```diff
@@ -106,12 +111,7 @@ public Object visit(ASTFunctionNode node, Object data) {
 
     @Override
     public Object visit(ASTReference node, Object data) {
```

Comment:
should we drop this method ?  Are reference nodes still being used just not in the same places?  If so what are they used for.

Reply:
Looking at the available NodeTransformRules, there aren't any that operate on reference nodes, so we can remove this.  

From what I have seen reference nodes are still used in certain situations, for instance with methods and identifier access nodes.  (e.g. something.someMethod(), and something.someProperty).  Identifier access nodes are a new concept with jexl3 and are used when using ant-style fields (i.e. fields with periods in them).